translate/storage/properties.py

```diff
@@ -478,11 +479,8 @@ def encoding(self):
             return self.personality.default_encoding
 
     def __str__(self):
-        """Convert to a string. Double check that unicode is handled
-        somehow here."""
-        source = self.getoutput()
-        assert isinstance(source, six.text_type)
-        return source.encode(self.encoding)
```

Comment:
In this particular format `self.encoding` might be referring to the personality encoding. See `@encoding` a few lines below.